src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -314,7 +314,9 @@ HARDWARE_INTRINSIC(Sve2,          BitwiseSelect,
 HARDWARE_INTRINSIC(Sve2,          BitwiseSelectLeftInverted,                                        -1,      3,     {INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_sve_bsl1n,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_HasRMWSemantics)
 HARDWARE_INTRINSIC(Sve2,          BitwiseSelectRightInverted,                                       -1,      3,     {INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_sve_bsl2n,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_HasRMWSemantics)
 HARDWARE_INTRINSIC(Sve2,          ShiftLeftAndInsert,                                               -1,      3,     {INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_sve_sli,        INS_invalid,        INS_invalid},     HW_Category_ShiftLeftByImmediate,  HW_Flag_Scalable|HW_Flag_HasImmediateOperand|HW_Flag_HasRMWSemantics)
-#define LAST_NI_Sve2              NI_Sve2_ShiftLeftAndInsert
+HARDWARE_INTRINSIC(Sve2,          Xor,                                                              -1,      3,     {INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_sve_eor3,       INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_SpecialCodeGen|HW_Flag_HasRMWSemantics)
```

Comment:
remind me, but why we need `HW_Flag_SpecialCodeGen` for this one? Did you try without adding this flag and what problems we run into?